src/cryptography/hazmat/primitives/serialization/ssh.py

```diff
@@ -757,6 +762,10 @@ def _serialize_ssh_private_key(
     nkeys = 1
     checkval = os.urandom(4)
     comment = b""
+    if hasattr(encryption_algorithm, "_comment"):
+        _comment_attr = getattr(encryption_algorithm, "_comment")
+        if isinstance(_comment_attr, bytes):
+            comment = _comment_attr
```

Comment:
I'm not sure what this is for, or where the comment attribute is supposed to come from?

Reply:
ok, we can omit checking isinstance of bytes